.github/workflows/ci.yml

```diff
@@ -248,7 +248,7 @@ jobs:
           - target: debian-x86_64
             os: ubuntu-latest
           - target: x86_64-apple-darwin
-            os: macos-12  # x64 CPU
+            os: macos-14  # M1 CPU
```

Comment:
You don't want M1 CPU here because we are building x86 target

Reply:
Oh ok. I thought we can build x86 target on M1, can we?

Reply:
I thought we could generate x86 and arm target both on M1, I'm not sure.

Reply:
Oh I see I see.

Reply:
Seems M1 could build and run(by Rosetta) `x86_64-apple-darwin`, but it's not the best choice. I've already switch back to macos-12 runner. Thx @nyurik